From ddfb2104372eba33a4b9701d08c190ad57151819 Mon Sep 17 00:00:00 2001
From: dineshsv <dineshsv@ami.com>
Date: Fri, 1 Sep 2023 18:42:28 +0530
Subject: [PATCH] Fix for EVB build error for OCP, Below are the driver
 changes.

 - pwm_Tach0
 - jtag0
 - jtag1
 - mctp
 - lpc_sio
 - i3c0
 - i3c2
 - i3c1

these are the driver changes for fixing OCP EVB build.

Signed-off-by: dineshsv <dineshsv@ami.com>
---
 arch/arm/boot/dts/aspeed-ast2600-evb.dts | 98 ------------------------
 1 file changed, 98 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-ast2600-evb.dts b/arch/arm/boot/dts/aspeed-ast2600-evb.dts
index 14f9577f3d95..1e024e3c8a22 100644
--- a/arch/arm/boot/dts/aspeed-ast2600-evb.dts
+++ b/arch/arm/boot/dts/aspeed-ast2600-evb.dts
@@ -115,64 +115,6 @@ &syscon {
 				  0x08 0x04>;		
 };
 
-&pwm_tacho {
-        status = "okay";
-        #pwm-cells = <3>;
-        aspeed,pwm-outputs = <7>;
-        pinctrl-names = "default";
-        pinctrl-0 = <&pinctrl_pwm0_default &pinctrl_tach0_default
-                        &pinctrl_pwm1_default &pinctrl_tach1_default
-                        &pinctrl_pwm2_default &pinctrl_tach2_default
-                        &pinctrl_pwm3_default &pinctrl_tach3_default
-                        &pinctrl_pwm4_default &pinctrl_tach4_default
-                        &pinctrl_pwm5_default &pinctrl_tach5_default
-                        &pinctrl_pwm12g1_default &pinctrl_tach6_default
-                        &pinctrl_pwm13g1_default &pinctrl_tach7_default
-                        &pinctrl_pwm14g1_default &pinctrl_tach8_default
-                        &pinctrl_pwm15g1_default &pinctrl_tach9_default
-                        &pinctrl_pwm7_default>;
-
-        fan@0 {
-                reg = <0x00>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x00>;
-        };
-        fan@1 {
-                reg = <0x01>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x01>;
-        };
-        fan@2 {
-                reg = <0x02>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x02>;
-        };
-        fan@3 {
-                reg = <0x03>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x03>;
-        };
-        fan@4 {
-                reg = <0x04>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x04>;
-        };
-        fan@5 {
-                reg = <0x05>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x05>;
-        };
-        fan@6 {
-                reg = <0x0c>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x06>;
-        };
-        fan@7 {
-                reg = <0x0d>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x07>;
-        };
-        fan@8 {
-                reg = <0x0e>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x08>;
-        };
-        fan@9 {
-                reg = <0x0f>;
-                aspeed,fan-tach-ch = /bits/ 8 <0x09>;
-        };
-};
 
 &kcs1 {
         aspeed,lpc-io-reg = <0xCA0>;
@@ -197,17 +139,7 @@ &lpc {
     status = "okay";
 };
 
-&jtag0 {
-        status = "okay";
-};
-
-&jtag1 {
-        status = "okay";
-};
 
-&mctp {
-    status = "okay";
-};
 
 &lpc_reset {
         status = "okay";
@@ -215,9 +147,6 @@ &lpc_reset {
 	pinctrl-0 = <&pinctrl_lpc_default>;
 };
 
-&lpc_sio {
-        status = "okay";
-};
 
 &lpc_snoop {
         snoop-ports = <0x80>;
@@ -258,7 +187,6 @@ ethphy2: ethernet-phy@0 {
 
 &mdio3 {
 	status = "okay";
-
 	ethphy3: ethernet-phy@0 {
 		compatible = "ethernet-phy-ieee802.3-c22";
 		reg = <0>;
@@ -436,32 +364,6 @@ &uart5 {
 	status = "okay";
 };
 
-&i3c0 {
-        status = "okay";
-        jdec-spd;
-        /* IDT IMX3102 multiplexer w/ thermal sensor */
-        imx3102: i3cmux@0f,3C000000000 {
-                reg = <0x0f 0x3C0 0x00000000>;
-                assigned-address = <0x0f>;
-        };
-
-        /* Renesas SPD5118 */
-        spd5118: spd@50,3C000000001 {
-                reg = <0x50 0x3C0 0x00000001>;
-                assigned-address = <0x50>;
-        };
-};
-&i3c2 {
-	status = "okay";
-    i3c_0: mipi_i3c@0,7ec05031000 {
-	compatible = "i3c-ibi-mqueue";
-	reg = <0x0 0x7ec 0x05031000>;
-	};	
-};
-&i3c1 {
-	status = "okay";
-	secondary;
-};
 
 &i2c0 {
 	status = "okay";
-- 
2.25.1

